src/default-options.ts

```diff
@@ -15,5 +15,6 @@ export default {
   },
   beforeMiddleware: [],
   afterMiddleware: [],
-  beforeStart: []
+  beforeStart: [],
+  initialize: null
```

Comment:
@anastasios-workable This overlaps with beforeStart. Initialize runs after diamorphosis while beforeStart is another queued function that is run after middlewares are initialized. Wdyt?

Reply:
TLDR I ported orka into cerberus and beforeStart was not enough

Reply:
I really think that diamorphosis should be moved out, it complicates things a lot and is not that much boilerplate tbh. Thoughts?

Reply:
I would rather not.  However, apps that would need initialize can call diamorphosis outside orka. Tested this in cerberus so I will drop this pr.